docs/example_go.md

```diff
@@ -18,13 +18,11 @@ We'll use Tilt to:
 - Measure the time from a code change to a new process
 - Optimize that time for fast feedback
 
-All the code is in this repo:
+All the code is in this repo: [tilt-example-go](https://github.com/windmilleng/tilt-example-go)
+{:.link-own-line}
```

Comment:
Putting this in one paragraph, and then using css to move it into multiple paragraphs, seems very semantically odd.

Have you considered doing this with pure CSS and not making any text changes at all?

e.g.,

```
p > a:only-child {
  text-indent: 2em;
  font-weight: bold;
}
```

Reply:
I considered that but the main thing I'm trying to do is get rid of the extra whitespace between the link and the paragraph above, so it's clear what bit of text the link "belongs" to; and messing with negative margins seemed foolhardy if say, the paragraph margins change from desktop to mobile. if the margins are consistent enough though, I'd happily do
```
p > a:only-child {
  text-indent: 2em;
  font-weight: bold;
  margin-top: -1em;
}
```
or whatever.

Reply:
ya, i like that suggestion. we typically set margin-bottom to $spacing-unit / 2, so maybe do margin-top: - $spacing-unit / 4

Reply:
cool, done.